src/CommonAPI/DBus/DBusMessage.cpp

```diff
@@ -292,5 +292,9 @@ bool DBusMessage::isSignalType() const {
     return (getType() == Type::Signal);
 }
 
+void DBusMessage::setNoReplyExpected(bool replyNotExpected) {
+    dbus_message_set_no_reply (message_, (0 != replyNotExpected) ? TRUE : FALSE);
```

Comment:
This is very convoluted.  
Set "No reply", using the variable named "Not Expected" by checking if it is  "Not equal to zero (false)" (i.e true)...

If you're sure that dbus_message_set_no_reply can use the bool type value directly then maybe just put it in directly?   If not, calling the values TRUE and FALSE do not make it any safer.  I would then pass in an integer value if you are unsure.

On the other hand, I would rename all of it to setReplyExpected and let the parameter be true if reply expected.